.github/workflows/update.yml

```
@@ -0,0 +1,39 @@
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./tools/version-checker
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node v16
+        uses: actions/setup-node@v2
```

actionlint (1.7.4)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)